### PR TITLE
Add Paramount+

### PR DESCRIPTION
**Pull request by fletcherdodd**

### CODE REVIEW

File: entries/p/paramountplus.com.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+			"sk",
+			"si",
+			"kr",
+			"es",
+			"se",
+			"ch",
+			"gb",
+			"us",
+			"uy",
+			"ve"
```
Comment (Carlgo11) [inline entries/p/paramountplus.com.json]: I only see that they support the following countries:
```suggestion
      "ar",
      "au",
      "bo",
      "br",
      "cl",
      "co",
      "cr",
      "de",
      "do",
      "ec",
      "fi",
      "gt",
      "hn",
      "ni",
      "no",
      "pa",
      "pe",
      "py",
      "se",
      "sv",
      "uy",
      "ve"
```
source: https://www.paramountplus.com/global/
screenshot

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Are you using tabs? I don't think JSON allows tabs :thinking:

File: entries/p/paramountplus.com.json (hunk 1 above)
Comment (Carlgo11) [reply entries/p/paramountplus.com.json]: @fletcherdodd you marked this as resolved without updating any code :thinking:

Comment (Carlgo11) [reply entries/p/paramountplus.com.json:61]: @fletcherdodd you marked this as resolved without updating any code :thinking:

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**